src/Moq/Linq/ConstructorCallVisitor.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Moq.Properties;
```

Comment:
* Please add the copyright notice at the beginning of the file. (See any other source code file for guidance.)
* Please put `System.*` imports before any `Moq.*` imports (and separate the two import groups by a blank line).